objects.js

```diff
@@ -2783,10 +2801,11 @@ SpriteMorph.prototype.reporterize = function (expressionString) {
             '*': 'reportProduct',
             '/': 'reportQuotient',
             '%': 'reportModulus',
-            'random': 'reportRandom'
+            'random': 'reportRandom',
+            'round': 'reportRound'
```

Comment:
I moved round there because even though it is monadic, it is a different type of block from the other monads, and it is easier to treat it as a diadic function. I should change the comments to make this clear